polaris-server.yml

```diff
@@ -90,7 +90,7 @@ defaultRealms:
 metaStoreManager:
   type: in-memory
   # type: eclipse-link # uncomment to use eclipse-link as metastore
-  # persistence-unit: polaris
```

Comment:
I intentionally keep this to be polaris rather than polaris-dev. polaris-dev is used in test and polaris is used for production.

Reply:
Otherwise, if we have persistence.xml from test resource and production resource, we are loading the one from test resource.

Reply:
Got it. Let me revert this one...but I don't think we will need this PR as the change is intensional based off https://github.com/polaris-catalog/polaris/pull/114/files. I think the only thing that is missing there is h2 lib import. Let me close this one and use 114 to add that in.

Reply:
@collado-mike close this one and added comment in your PR for including h2 lib if people ant to use default persist backend with h2.